nnpdf_data/nnpdf_data/filter_utils/star/jet_dijet_2009_200gev.py

```diff
@@ -0,0 +1,322 @@
+"""This script provides the common filer to the jet and dijet STAR 2009 datasets.
+Files need to be parsed all together as there are correlations provided.
+"""
```

Comment:
Filters should not belong to the utils. This deprecates the purpose of the utils which in essence should contain the collections of functions that are share by various filters $-$ not the filters themselves. The main filters should be as close to the raw data as much as possible, hence why the structure needs to be `dataset/filter.py`.

In this regard, reading/parsing of rawdata and generation of the commondata files have to be moved to `dataset/filter.py` and only shared functions should be kept here.

Reply:
I agree storing them inside the utils is not the best location, but such filters are common for multiple datasets. 
I followed what was done for the EIC utils...  
We can put them top level in `nnpdf_data/nnpdf_data/` if this is cleaner for you.

Reply:
done in e3805a787